Objects/object.c

```diff
@@ -2107,6 +2107,12 @@ _PyTrash_thread_destroy_chain(void)
          * up distorting allocation statistics.
          */
         assert(op->ob_refcnt == 0);
+        /* The deallocator executing this function still occupies
```

Comment:
What I want to say is the "root deallocator" is still doing job and the stack is not released yet. So counting it into trash_delete_nesting is natural. And I also want to list the example you give here. Is it okay?